core/tests/protractor/parameters.js

```diff
@@ -0,0 +1,96 @@
+// Copyright 2014 The Oppia Authors. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS-IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+/**
+ * @fileoverview End-to-end tests of parameters / expressions.
+ */
+
+var general = require('../protractor_utils/general.js');
+var forms = require('../protractor_utils/forms.js');
+var users = require('../protractor_utils/users.js');
+var workflow = require('../protractor_utils/workflow.js');
+var editor = require('../protractor_utils/editor.js');
+var player = require('../protractor_utils/player.js');
+
+describe('Parameters', function() {
```

Comment:
Can you add the file `protractor/parameters.js` to the `editorFeatures` suite in `protractor.conf.js`?

See https://github.com/oppia/oppia/blob/develop/core/tests/protractor.conf.js#L68